.github/workflows/build-push.yml

```diff
@@ -113,22 +120,24 @@ jobs:
 
       - name: Build and push release
         if: endsWith(github.event.ref, '-scylla')
-        uses: docker/build-push-action@v3
+        uses: docker/build-push-action@v4
         with:
           context: .
           file: ./Dockerfile
+          platforms: linux/amd64,linux/arm64
           push: true
           tags: |
             scylladb/scylla-cqlsh:${{ env.DOCKER_TAG }}
             scylladb/scylla-cqlsh:latest
 
       - name: Build and push master
-        uses: docker/build-push-action@v3
-        if: endsWith(github.event.ref, 'master')
+        uses: docker/build-push-action@v4
+        # if: endsWith(github.event.ref, 'master')
```

Comment:
Don't you want to remove these comment lines?

Reply:
No.

It was done only for testing (by mistake it wasn't on a separate commit)